app/controllers/discover.js

```diff
@@ -179,50 +179,18 @@ export default ApplicationController.extend({
         this.get('debouncedLoadPage')();
     },
 
-    addFilters() {
-        var filters = this.get('facetFilters');
-        let params = this.get('filterQueryParams');
-        let associations = this.get('associationFilters');
-        for (var param in params) {
-            let key = params[param];
-            if (params.indexOf(key) > -1) {
-                let filterValue = this.get(key);
-                if (filterValue) {
-                    let filter = {};
-                    if (associations.indexOf(key) > -1) {
-                        filter = associationTermsFilter(key, filterValue.split(','));
-                    } else if (key === 'contributors') {
-                        filter = personTermsFilter(key, filterValue.split(','));
-                    } else if (key === 'sources') {
-                        filter = termsFilter(key, filterValue.split(','), false);
-                    } else if (key === 'type') {
-                        filter = termsFilter('@type', filterValue.split(','));
-                    } else {
-                        filter = termsFilter(key, filterValue.split(','));
-                    }
-                    filters.set(key, filter);
-                }
-            }
-        }
-        if (this.get('start') && this.get('end')) {
-            let filter = dateRangeFilter('date', this.get('start'), this.get('end'));
-            filters.set('date', filter);
-        }
-        this.send('filtersChanged', filters);
-    },
-
     facets: Ember.computed(function() {
         return [
-            { key: 'sources', title: 'Source', type: 'source', component: 'search-facet-source', raw: false },
-            { key: 'date', title: 'Date', component: 'search-facet-daterange' },
-            { key: '@type', title: 'Type', component: 'search-facet-worktype' },
-            { key: 'tags', title: 'Subject/Tag', component: 'search-facet-typeahead', type: 'tag', raw: true },
-            { key: 'publisher', title: 'Publisher', component: 'search-facet-association' },
-            { key: 'funder', title: 'Funder', component: 'search-facet-association' },
-            { key: 'institution', title: 'Institution', component: 'search-facet-association' },
-            { key: 'organization', title: 'Organization', component: 'search-facet-association' },
-            { key: 'language', title: 'Language', component: 'search-facet-language' },
-            { key: 'contributors', title: 'People', type: 'person', useId: true, component: 'search-facet-person' },
+            { key: 'sources', title: 'Source', type: 'source', component: 'search-facet-source', raw: false, param: this.get('sources'), facetFilters: this.get('facetFilters') },
```

Comment:
>
Could we use this param as the facet's state?
>

That would be ideal. What would that look like on the component side? Passing `param` instead of `filter` in via `faceted-search`?